styleguide/_themes/global/scss/banners.scss

```diff
@@ -36,31 +36,147 @@ $banner-text-margin-medium: 10%;
 
 .banner-headline {
   @include heading($white);
-  font-size: 3em;
-  line-height: 1em;
+  font-size: 4rem;
+  line-height: 3.75rem;
   text-align: center;
 }
 
 
-.imacBanner-container {
-  background-color: $brand-primary;
+.imacBanner {
+  background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/bananers/backgrounds/o365-overview.jpg');
+  background-size: cover;
   color: $white;
+  margin-top: 50px;
+  overflow: hidden;
 
-  &.large {
+  &.small {
+    height: 250px;
+    padding-top: 50px;
+  }
+
+  &.medium {
     height: 300px;
-    padding-top: 30px;
+    padding-top: 50px;
+  }
+
+  &.large {
+    height: 350px;
+    padding-top: 50px;
   }
 }
 
+.imacBanner-container {
+  @include make-container;
+  @include fade(.25s, ease-in-out);
+  opacity: 0;
+}
+
+.wf-khand-n7-active {
+  .imacBanner-container {
+    opacity: 1;
+  }
+}
+
+.imacBanner-textRow {
+  @include make-row;
+}
+
+.imacBanner-headlineLarge {
+  @include make-sm-column(7);
+}
+
+.imacBanner-headlineMedium {
+  @include make-sm-column(6);
+}
+
+.imacBanner-headlineSmall {
+  @include make-sm-column(5);
+}
+
 .imacBanner-headline {
   @include heading($white);
+  font-family: 'Khand';
```

Comment:
It probably makes sense to define this font as a `$imac-font` variable or something like that, and place it in `styleguide/_themes/global/scss/fonts.scss` instead of just explicitly writing it out here.